udata/core/spatial/models.py

```diff
@@ -116,7 +116,7 @@ def child_level(self):
         HANDLED_LEVELS = current_app.config.get('HANDLED_LEVELS')
         try:
             return HANDLED_LEVELS[HANDLED_LEVELS.index(self.level) - 1]
-        except IndexError:
+        except (IndexError, ValueError):
```

Comment:
When does it happen? Maybe `current_app.config.get('HANDLED_LEVELS', tuple())` is more explicit if you try to address that case?